setup.py

```diff
@@ -20,10 +21,17 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
+version_namespace = {}
+version_path = convert_path('bleach/version.py')
+with open(version_path) as version_file:
+    exec(version_file.read(), version_namespace)
```

Comment:
I'm not wild about doing an `exec` in `setup.py`. Generally how I deal with this is by parsing out the line with `__version__` in it.

https://github.com/willkg/dennis/blob/master/setup.py#L19

Can you redo this using the same method? The version can stay in `bleach/version.py` if you want--I don't really care so much about that aspect.